src/traces/scatter/attributes.js

```diff
@@ -389,6 +401,37 @@ module.exports = {
             anim: true,
             description: 'Sets the marker opacity.'
         },
+        angle: {
+            valType: 'angle',
+            dflt: 0,
+            arrayOk: true,
+            editType: 'plot',
+            anim: false, // TODO: possibly set to true in future
+            description: [
+                'Sets the marker angle in respect to `angleref`.'
+            ].join(' ')
+        },
+        angleref: {
+            valType: 'enumerated',
+            values: ['previous', 'up'],
+            dflt: 'up',
+            editType: 'plot',
+            anim: false,
+            description: [
+                'Sets the reference for marker angle.',
+                'With *previous*, angle 0 points along the line from the previous point to this one.',
+                'With *up*, angle 0 points toward the top of the screen.'
+            ].join(' ')
+        },
+        standoff: {
+            valType: 'number',
+            min: 0,
+            dflt: 0,
+            arrayOk: true,
+            editType: 'plot',
+            anim: true,
+            description: 'Sets the marker standoff in the direction of `angle` (in px).'
```

Comment:
```suggestion
            description: [
                'Moves the marker away from the data point in the direction of `angle` (in px).',
                'This can be useful for example if you have another marker at this',
                'location and you want to point an arrowhead marker at it.'
            ]
```